### PR TITLE
:sparkles: Add an option to provide content

### PR DESCRIPTION
**Pull request by jtanguy**
Using a simple require, we can exploit handlebars templates in the cli too.

Since templating is baked into the node lib, and some puppeteer options can be passed from environment variables, this cli has a lot of uses in itself.

### CODE REVIEW

**Comment by frinyvonnick** [comment]
Thank you for your contribution @jtanguy 🙏 

Could you provide an example of usage please?

**Comment by jtanguy** [comment]
Here I added some examples in the repo.
However, by using the tool yesterday, I forgot to put an image file name, and overwrote my data file with the contents of the image.

Somehow yargs parse the content option of  `example/handlebars.html --content=example/handlebars.json` as example/handlebars.json and thus the file is overwritten :/

File: README.md [hunk 1]
```diff
@@ -39,8 +39,16 @@ npm install -g node-html-to-image-cli
...
+
+```sh
+npx node-html-to-image-cli examples/simple.html ./image.png
+```
+
+With additional data passed to the template
+
 ```sh
-npx node-html-to-image-cli ./index.html ./image.png
+npx node-html-to-image-cli examples/handlebars.html ./image.png --content=./examples/handlebars.json
```
Comment (frinyvonnick) [inline README.md]:
```suggestion
npx node-html-to-image-cli examples/handlebars.html ./image.png --content ./examples/handlebars.json
```

**Review by frinyvonnick — APPROVED** [review]

**Comment by frinyvonnick** [comment]
Options in yargs are called with equal character:

```
--content example/handlebars.json
```

Looks good to me @jtanguy!

File: README.md (hunk 1 above)
Comment (jtanguy) [reply README.md]: I was using equals tough, if I recall correctly

File: examples/handlebars.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-size: 5vh;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Hello {{you}}</h1>
```
Comment (frinyvonnick) [inline examples/handlebars.html:22]: I would prefer `name` rather than `you` with `"name": "you"` in the `handlebars.json` file. What do you think of it?

**Comment by frinyvonnick** [comment]
I tested it. I works with and without an equal as well!

File: README.md [hunk 3]
```diff
@@ -39,8 +39,16 @@ npm install -g node-html-to-image-cli
...
+
+```sh
+npx node-html-to-image-cli examples/simple.html ./image.png
+```
+
+With additional data passed to the template
+
 ```sh
-npx node-html-to-image-cli ./index.html ./image.png
+npx node-html-to-image-cli examples/handlebars.html ./image.png --content ./examples/handlebars.json
```
Comment (frinyvonnick) [inline README.md:51]: When users install the cli they don't have the examples folder. I prefer it doesn't appear in the usage examples.

File: examples/handlebars.html (hunk 2 above)
Comment (jtanguy) [reply examples/handlebars.html:22]: As you wish

File: README.md (hunk 3 above)
Comment (jtanguy) [reply README.md:51]: The same could be said of `index.html`in the user's current directory. 

We can either inline the examples or  mention that the examples folder is located in the repository

Comment (frinyvonnick) [reply README.md:51]: A mention with a link to the examples folder would be nice 👍

Comment (frinyvonnick) [reply README.md]: I can add it after if you want @jtanguy ?

**Comment by frinyvonnick** [comment]
@jtanguy I can deal with my comments in a futher pull request if you prefer?

**Comment by jtanguy** [comment]
Maybe yes, I don't have much time at the moment

**Pull request merged**